.devcontainer/java/devcontainer.json

```diff
@@ -0,0 +1,57 @@
+{
+    "name": "Java",
+    "image": "mcr.microsoft.com/devcontainers/java:17",
+    "workspaceFolder": "/workspaces/${localWorkspaceFolderBasename}/server/java",
+    "forwardPorts": [
+        8080
+    ],
+    "portsAttributes": {
+        "8080": {
+            "label": "Java",
+            "onAutoForward": "openBrowserOnce"
+        }
+    },
+    "secrets": {
+        "BRAINTREE_MERCHANT_ID": {
+            "description": "Sandbox merchant ID of the application.",
+            "documentationUrl": "https://www.braintreepayments.com/sandbox"
+        },
+        "BRAINTREE_PUBLIC_KEY": {
+            "description": "Sandbox public key of the application.",
+            "documentationUrl": "https://www.braintreepayments.com/sandbox"
+        },
+        "BRAINTREE_PRIVATE_KEY": {
+            "description": "Sandbox private key of the application.",
+            "documentationUrl": "https://www.braintreepayments.com/sandbox"
+        }
+    },
```

Comment:
I believe for SDK's, don't need to setup the BASE_URL. If environment is configured as sandbox, it will have all these configurations.